node/test_helpers.go

```diff
@@ -117,16 +117,21 @@ func waitForAtLeastNBlocks(node Node, n int, source Source) error {
 	})
 }
 
-// TODO: use n and return n validators
 func getGenesisValidatorSetWithSigner(n int) ([]cmtypes.GenesisValidator, crypto.PrivKey) {
 	nodeKey := &p2p.NodeKey{
 		PrivKey: genesisValidatorKey,
 	}
 	signingKey, _ := GetNodeKey(nodeKey)
 	pubKey := genesisValidatorKey.PubKey()
 
```

Comment:
Building on the coderabbitai comment, all of this should be within the for loop so that validators have unique keys.

Reply:
Removed the for loop as we will only return one validator.